when-shit-hits-the-fan.adoc

```diff
@@ -150,52 +150,52 @@ should and can be handled differently, it is not always so clear cut.
 It's prudent to treat things as more severe than you know them to be.
 ____
 
-So even if a vulnerability seems hard to exploit, it might be best to
-assume it's easily exploited and that you just haven't figured out
+So, even if a vulnerability seems hard to exploit, it might be best to
+assume that it's easily exploitable and you just haven't figured out
 how yet.
 
-He also mentions how "it's somewhat incorrect to call this thread
+He also mentions how "`it's somewhat incorrect to call this thread
 anything about disclosure, this thread is not about
 disclosure. Disclosure is when you tell the vendor.  This thread is
 about publication and that has very different
 implications. Publication is when you're sure you've told the
-prospective attackers." This last observation of the distinction
+prospective attackers`". This last observation concerning the distinction
 between disclosure and publication is an important one. The easy part
-is responsible disclosure; the hard part is to sensibly publish.
+is responsible disclosure; the hard part is sensible publishing.
 
 === Traumatic childhood
 
 Bitcoin started out as a one-man (at least that's what its creator's
-pseudonym suggests) project, and bitcoin had little to no value. As
+pseudonym suggests) project, and bitcoin had initially little to no value. As
 such, vulnerabilities and bug fixes were not as rigorously handled as
 they are today.
 
-The Bitcoin wiki has
-https://en.bitcoin.it/wiki/Common_Vulnerabilities_and_Exposures[a list
+The Bitcoin wiki has a 
+https://en.bitcoin.it/wiki/Common_Vulnerabilities_and_Exposures[list
 of common vulnerabilities and exposures]
 (CVEs)
-that Bitcoin has gone through. This section is a little exposé of some
+that Bitcoin has gone through. This section constitutes a little exposé of some
 of the security issues and incidents from the early years of
-Bitcoin. We won't cover them all, but we have selected a few that we
-find extra interesting.
+Bitcoin. We won't cover them all, but we selected a few that we
+find extremely interesting.
```

Comment:
extremely sounds too extreme. How about "quite", "very", or "especially"

Reply:
"Especially" is the most appropriate imo